index.html

```diff
@@ -2,9 +2,168 @@
 <html>
 <head>
 	<title>Sandwich Maker</title>
+	<link rel="stylesheet" type="text/css" href="Main.css">
 </head>
+
 <body>
+	<h2> Delicious Sandwich </h2>
+	<div id = "container" >
+
+		<form id= "breadForm" class="form">
+			<legend> Bread </legend>
+			<div id = "bread-chooser">
+				<label for="italian" > 
+					<input type="radio" name="bread" value="italian" id="italian"> Italian
+				</label>
+
+				<label for="flatbread" > 
+					<input type = radio name="bread" value="flatbread" id="flatbread"> Flat Bread
```

Comment:
You are leaving off the quotes.  The browser is still interpreting it correctly but you are playing with fire. 😃 `<input type="radio"`